shell.nix

```diff
@@ -0,0 +1,10 @@
+{ pkgs ? import <nixpkgs> {} }:
```

Comment:
I would pin this to a specific commit to avoid unpredictable behavior, like we do in other repos:
https://github.com/status-im/status-go/blob/53ac61bb8bf9c3356ee2d65e959ae314b3ffbda3/shell.nix#L3-L7

Maybe even using the same commit is a good idea.